### PR TITLE
Added that vic_machine_server runs as service in appliance

### PR DESCRIPTION
**Pull request by stuclem**
- Fixes https://github.com/vmware/vic-product/issues/947
- Fixes https://github.com/vmware/vic-product/issues/923

@zjs is this OK? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The following services run in the vSphere Integrated Containers appliance:
 - vSphere Integrated Containers Registry service
 - vSphere Integrated Containers Management Portal service
 - The file server for vSphere Integrated Containers Engine downloads and installation of the vSphere Client plug-ins
-- The `vic-machine` server service, that powers the Create Virtual Container Host wizard in the HTML5 vSphere Client plug-in
+- The `vic-machine_server` service, that powers the Create Virtual Container Host wizard in the HTML5 vSphere Client plug-in
```
Comment (zjs) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: > `vic-machine_server`

If this is referring to the systemd service entry, this should be `vic_machine_server` (with two underscores). If it's referring to the server binary or container, it should be `vic-machine-server` (with two hyphens).

Eventually, we should give this a "real name", but it'd probably end up being something like "vSphere Integrated Containers Virtual Container Host Management API Server", which... seems really verbose.

File: docs/user_doc/vic_vsphere_admin/restart_services.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ You can restart the vSphere Integrated Containers services that run in the appli
 - vSphere Integrated Containers Registry service
 - vSphere Integrated Containers Management Portal service
 - The file server for vSphere Integrated Containers Engine downloads and installation of the vSphere Client plug-ins
-- The `vic-machine` server service, that powers the Create Virtual Container Host wizard in the HTML5 vSphere Client plug-in
+- The `vic-machine_server` service, that powers the Create Virtual Container Host wizard in the HTML5 vSphere Client plug-in
```
Comment (zjs) [inline docs/user_doc/vic_vsphere_admin/restart_services.md]: Ditto

File: docs/user_doc/vic_vsphere_admin/restart_services.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ You deployed the vSphere Integrated Containers appliance.
...
 
-  - vSphere Integrated Containers Registry: `systemctl restart harbor.service`
-  - vSphere Integrated Containers Management Portal services: `systemctl restart admiral.service`
-  - Embedded file server: `systemctl restart fileserver.service`
-  - `vic-machine` server: `systemctl restart vic_machine_server.service`
\ No newline at end of file
+  - vSphere Integrated Containers Registry: <pre>systemctl restart harbor.service</pre>
+  - vSphere Integrated Containers Management Portal services: <pre>systemctl restart admiral.service</pre>
+  - Embedded file server: <pre>systemctl restart fileserver.service</pre>
+  - `vic-machine_server`: <pre>systemctl restart vic_machine_server.service</pre>
```
Comment (zjs) [inline docs/user_doc/vic_vsphere_admin/restart_services.md]: Ditto

File: docs/user_doc/vic_vsphere_admin/security_reference.md [hunk 4]
```diff
@@ -49,6 +49,7 @@ The vSphere Integrated Containers appliance makes the core vSphere Integrated Co
 |443|HTTPS|Connections to vSphere Integrated Containers Registry from vSphere Integrated Containers Management Portal, VCHs, and Docker clients|
 |4443|HTTPS|Connections to the Docker Content Trust service for vSphere Integrated Containers Registry|
 |8282|HTTPS|Connections to vSphere Integrated Containers Management Portal UI and API|
+|8443|HTTPS|Connections to the `vic-machine_server` service, that powers the Create Virtual Container Host wizard in the HTML5 vSphere Client plug-in|
```
Comment (zjs) [inline docs/user_doc/vic_vsphere_admin/security_reference.md]: Ditto

File: docs/user_doc/vic_vsphere_admin/service_status.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ You can check the status of the vSphere Integrated Containers services that run
 - vSphere Integrated Containers Registry service
 - vSphere Integrated Containers Management Portal service
 - The file server for vSphere Integrated Containers Engine downloads and installation of the vSphere Client plug-ins
-- The `vic-machine` server service, that powers the Create Virtual Container Host wizard in the HTML5 vSphere Client plug-in
+- The `vic-machine_server` service, that powers the Create Virtual Container Host wizard in the HTML5 vSphere Client plug-in
```
Comment (zjs) [inline docs/user_doc/vic_vsphere_admin/service_status.md]: Ditto

File: docs/user_doc/vic_vsphere_admin/service_status.md [hunk 6]
```diff
@@ -16,10 +16,10 @@ You deployed the vSphere Integrated Containers appliance.
...
 2. Run one of the following commands to check the status of one of the vSphere Integrated Containers services:
 
-  - vSphere Integrated Containers Registry: `systemctl status harbor.service`
-  - vSphere Integrated Containers Management Portal services: `systemctl status admiral.service`
-  - Embedded file server: `systemctl status fileserver.service`
-  - `vic-machine` server: `systemctl status vic_machine_server.service`
+  - vSphere Integrated Containers Registry: <pre>systemctl status harbor.service</pre>
+  - vSphere Integrated Containers Management Portal services: <pre>systemctl status admiral.service</pre>
+  - Embedded file server: <pre>systemctl status fileserver.service</pre>
+  - `vic-machine_server`: <pre>systemctl status vic_machine_server.service</pre>
```
Comment (zjs) [inline docs/user_doc/vic_vsphere_admin/service_status.md]: Ditto

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Let's stick with `vic_machine_server` for now :)

File: docs/user_doc/vic_vsphere_admin/restart_services.md (hunk 2 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/restart_services.md]: Fixed

File: docs/user_doc/vic_vsphere_admin/restart_services.md (hunk 3 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/restart_services.md]: Fixed

File: docs/user_doc/vic_vsphere_admin/security_reference.md (hunk 4 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/security_reference.md]: Fixed

File: docs/user_doc/vic_vsphere_admin/service_status.md (hunk 5 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/service_status.md]: Fixed

File: docs/user_doc/vic_vsphere_admin/service_status.md (hunk 6 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/service_status.md]: Fixed. Why does one only make typos in chunks of text that get copied and pasted into multiple locations?

**Comment by stuclem** [comment]
Thanks @zjs. All OK now?

**Pull request merged**